docs/bake-reference.md

```diff
@@ -832,6 +834,20 @@ RUN --mount=type=secret,id=KUBECONFIG \
     KUBECONFIG=$(cat /run/secrets/KUBECONFIG) helm upgrade --install
 ```
 
+### `target.shm-size`
```

Comment:
The docs should describe a bit about the feature as well, so what it does and how it applies to build containers, not only the formatting of the value. Same for ulimits that can define the values.

Maybe it also makes sense to mention that this is advanced customization that usually is not needed and should be let for builder to figure out automatically.